app/models/publisher.rb

```diff
@@ -239,6 +240,36 @@ def dont_destroy_verified_publishers
     throw :abort if verified?
   end
 
+  # Returns publishers who have abandoned verification process and do not have another publisher account
+  def self.get_winback_publishers
+    verified_publishers = Publisher.where(verified: true)
```

Comment:
At some point in the future loading all publishers into memory at once will not work. In the loop over `unverified_publishers` you could instead make a query to find a verified publisher by the unverified email.